### PR TITLE
Chore/merge main@8959bf3

### PR DESCRIPTION
**Pull request by julien-deramond**
- [x] https://github.com/twbs/bootstrap/commit/46bb9e78d74e0c7f4fa6f6fa611f3d9f8f28860b
- [x] https://github.com/twbs/bootstrap/commit/bb4f9d74fb399986eeeed07da10eded0908200b1
- ~~https://github.com/twbs/bootstrap/commit/e12e0806fbf7f7896316a142d732052a72b16aab~~
- ~~https://github.com/twbs/bootstrap/commit/da541014cb9caf830b77fa8993da0465b7210aac~~
- [x] https://github.com/twbs/bootstrap/commit/b5a956781f17a153626bb60ef09ef2a80dc74b8f
- ~~https://github.com/twbs/bootstrap/commit/416f4905f13c2ca3096343e0a5a4934c8c0ced3f~~
- [x] https://github.com/twbs/bootstrap/commit/73e45ee36cf4fdb6014267ecc67ba53a8b25fdae
- [x] https://github.com/twbs/bootstrap/commit/8959bf3f0bbaca514967661a6b721a3f065f49a9

### CODE REVIEW

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
Bug on nested nav scrollspy sm breakpoint:
image

**Comment by julien-deramond** [comment]
> Bug on nested nav scrollspy sm breakpoint:

This is not a regression from this branch; I can reproduce it directly in https://boosted.netlify.app/docs/5.2/components/scrollspy/. May worth creating an issue.

File: site/content/docs/5.2/examples/features/index.html [hunk 1]
```diff
@@ -136,7 +136,7 @@ <h2>Featured title</h2>
         </div>
       </div>
       <div class="col d-flex align-items-start">
-        <div class="icon-square bg-light text-dark d-inline-flex align-items-center justify-content-center fs-4 flex-shrink-0 me-3">
+        <div class="icon-square text-bg-light d-inline-flex align-items-center justify-content-center fs-4 flex-shrink-0 me-3">
           <svg class="bi" width="1em" height="1em"><use xlink:href="#tools"/></svg>
         </div>
         <div>
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/content/docs/5.2/examples/features/index.html:142]: Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1345/commits/18a552f270fa1d6e4c75b5eca2323619cac6299f

File: site/assets/scss/_masthead.scss [hunk 2]
```diff
@@ -17,15 +17,22 @@
...
-    // End mod
+
+    @include media-breakpoint-up(lg) {
+      padding-right: 4rem;
+    }
   }
   .btn-clipboard {
-    margin-top: .5rem;
+    position: absolute;
+    top: .3rem; // Boosted mod
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/assets/scss/_masthead.scss]: Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1345/commits/9a825ad0cfd1303ec90de028656b251e92c0ebb5

File: site/layouts/partials/home/masthead.html [hunk 3]
```diff
@@ -15,9 +15,9 @@ <h1 class="mb-3 display-1">Orange B<span class="text-primary">oo</span>sted<br>w
       <p class="lead mb-4">
         Orange Boosted is a Bootstrap based, Orange branded accessible and ergonomic components library.
       </p>
-      <div class="d-flex flex-column flex-md-row align-items-md-center justify-content-md-center gap-3 mb-4">
+      <div class="d-flex flex-column flex-lg-row align-items-md-center justify-content-md-center gap-3 mb-4">
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/layouts/partials/home/masthead.html]: Definitely better with `.align-items-md-center`! Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1345/commits/56f2796c01376767ac7530f6296a8141f47fbde1

**Review by louismaximepiton — APPROVED** [review]
LGTM 🚀

**Pull request merged**